src/main/cljc/jarl/eval.cljc

```diff
@@ -474,10 +474,16 @@
       (catch ExceptionInfo e
         (let [type (errors/ex-type e)]
           (if (or (= type :jarl.exceptions/builtin-exception) (= type :jarl.exceptions/type-exception))
-            (if (= name "regex.is_valid")
+            (condp = name
```

Comment:
Perhaps we could add the expected return value as an attribute on the exception 🤔 . That way, we can keep this per-builtin special knowledge closer to the implementation.

Reply:
Yeah. On the other hand, I kinda _want_ these exceptions to be annoying, as it increases the chances of these things getting fixed in OPA. The graph reachable ones looks like they can be removed from here not too long from now 😍 https://github.com/open-policy-agent/opa/pull/4956